### ISSUE TITLE
Merging Option: Only add missing fields from new entries

### ISSUE THREAD

**Issue by endfreude** [ChenglongMa/zoplicate#15]
It would be nice to have the following option for merging:
Use first added data BUT add missing fields from latest added data.

**Comment by endfreude** [comment]
My current use case:
Adding the abstact field from new data (which unfortunately does not contain the publication field)

**Comment by ChenglongMa** [comment]
Got it!

I will add this option in the next version.

Thanks for your feedback!

**Comment by Apollo7777777** [comment]
It seems that the plugin can't work in the new version of Zotero beta57 ？

**Comment by ChenglongMa** [comment]
Really? Thanks for pointing out this issue.

I'll check it soon.

Thanks.

**Comment by Apollo7777777** [comment]
sorry, it can work in the beta57.  I accidentally turned it off

**Comment by ChenglongMa** [comment]
No worries. Thanks for your attention :)

**Comment by ChenglongMa** [comment]
Hi @endfreude,

I've launched [v2.2.0](https://github.com/ChenglongMa/zoplicate/releases/tag/2.2.0) with the functions you suggested integrated. 

Instead of adding a new option, I implemented it universally: Regardless of the chosen option, the plugin now extracts information from duplicate items and fills in missing fields in the retained item.

Feel free to share any feedback or suggestions for further improvement.

Appreciate your valuable input 💖!

Best regards,
Chenglong
